Dockerfile.web-client

```diff
@@ -0,0 +1,29 @@
+# Use official Node.js image as the base image for web client
+FROM node:14 as web-client-builder
```

Comment:
Could we use a more recent version of `node`?

Reply:
yes, sure. done